### ISSUE TITLE
Lock the Evaluation Lane contract

### ISSUE THREAD

**Issue by HowBoring** [HowBoring/haidass-kev-train#8]
## Question

What is the exact Evaluation Lane contract for Kev v6 primary evaluation and Kev v4 regression evaluation? Lock checkpoint discovery, CPU/GPU scheduling, raw and temperature-scaled report schemas, accuracy/NLL/Brier/RPS/MAE metrics, option-reordering stability, calibration inputs, failure behavior, and the evidence required for SFT Complete.

**Comment by HowBoring** [comment]
Evaluation Lane contract is implemented and smoke-verified. Primary suite v6; regression v4; manifest checksums and case-group split separation are enforced/verified. Training evaluates the full development split and selects minimum macro NLL with equal question-src weighting, without temperature fitting. Reports include micro NLL, accuracy, Brier, ordinal normalized RPS and expected-index MAE, counts and source/type breakdowns. Final CLI: python -m haidass_kev_train.evaluation.run --artifact PATH --suite SUITE --split test --calibration-out PATH --out PATH --reorder-stability. Calibration uses only the separate calibration split and produces a separate artifact bound to portable-model hash and split identity; raw/calibrated reports are distinct. Choice reversal preserves hard/soft targets, excludes ordinal score. Nonfinite valid logits/invalid target mass fail. GPU eval and training serialize; CPU prep can overlap. Pilot CLI exercised all 1088 calibration decisions, 1468 development decisions and 756 choice reversals successfully; artifacts/reports/sft-pilot-evaluation.json is pipeline evidence, NOT a trained-model quality claim. Locked test remains unused. SFT Complete requires the fixed-budget selected checkpoint plus independent calibration and locked-test reports; four-step pilot is only runnable evidence.
